net.devolutions.RDMFree.yml

```diff
@@ -0,0 +1,42 @@
+app-id: net.devolutions.RDMFree
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: remotedesktopmanagerfree
+tags: [proprietary]
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --filesystem=~/.rdm:create
```

Comment:
Any reason this is not `--persist`?

Reply:
We have other products that expect ~/.rdm to exists if RDMFree was used.